src/main/kotlin/org/wycliffeassociates/otter/jvm/recorder/device/AudioRecorder.kt

```diff
@@ -0,0 +1,85 @@
+package org.wycliffeassociates.otter.jvm.recorder.device
+
+import io.reactivex.Observable
+import io.reactivex.schedulers.Schedulers
+import io.reactivex.subjects.PublishSubject
+import org.wycliffeassociates.otter.common.device.IAudioRecorder
+import javax.sound.sampled.AudioFormat
+import javax.sound.sampled.AudioSystem
+
+class AudioRecorder : IAudioRecorder {
+
+    private val monitor = Object()
+
+    @Volatile
+    private var stop = false
+
+    companion object {
+        val SAMPLE_RATE = 44100F // Hz
+        val SAMPLE_SIZE = 16 // bits
+        val CHANNELS = 1
+        val SIGNED = true
+        val BIG_ENDIAN = false
+        val FORMAT = AudioFormat(
+            SAMPLE_RATE,
+            SAMPLE_SIZE,
+            CHANNELS,
+            SIGNED,
+            BIG_ENDIAN
+        )
+        val BUFFER_SIZE = 1024
+    }
+
+    private var line = AudioSystem.getTargetDataLine(FORMAT)
+    private val audioByteObservable = PublishSubject.create<ByteArray>()
+    private val recordingStream = Observable.fromCallable {
```

Comment:
If you put `fromCallable` on the next line, and `subscribeOn` on its own line below, then everything will be aligned nicely. https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping